### PR TITLE
feat(1314): Add description of --local option to sd-local

### PR DESCRIPTION
**Pull request by kkisic**
## Context
Add description of `--local` option to sd-local.

## References
https://github.com/screwdriver-cd/screwdriver/blob/master/design/sd-local.md#options-1

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by yuichi10 — APPROVED** [review]

**Review by kkokufud — APPROVED** [review]
LGTM

File: docs/user-guide/local.md [hunk 1]
```diff
@@ -160,6 +167,7 @@ The following options can be used with the build command:
 |--artifacts-dir|The build artifacts destination (default: `./sd-artifacts`)|
 |-e, --env|The environment variables in a build environment using `<key>=<value>` format (multiple variables allowed)|
 |--env-file|The environment variables in a build environment using file format (refer to [env-file format](#env-file-format))|
+|--local|Run command with .sdlocal/config file in current directory.|
```
Comment (tkyi) [inline docs/user-guide/local.md]:
```suggestion
|--local|Run command with `.sdlocal/config` file in current directory.|
```

File: docs/user-guide/local.md [hunk 2]
```diff
@@ -124,6 +124,13 @@ You can confirm the current settings.
 $ sd-local config view
 ```
 
+## Options
+The following options can be used with the config command:
+
+
+|option|description|
+|---|---|
+|--local|Read/Modify .sdlocal/config file in current directory|
```
Comment (tkyi) [inline docs/user-guide/local.md]:
```suggestion
|--local|Read/Modify `.sdlocal/config` file in current directory|
```

**Review by tkyi — COMMENTED** [review]
Small suggestion

**Comment by yuichi10** [comment]
@tkyi Thank you for your suggestion!

**Review by tk3fftk — APPROVED** [review]
LGTM:+1:

**Review by tkyi — APPROVED** [review]

**Pull request merged**